lib/rucio/api/replica.py

```diff
@@ -107,20 +108,24 @@ def declare_bad_file_replicas(replicas, reason, issuer, vo='def', session=None):
                 "rse_id": rse_id,
                 "name": r["name"]
             })
-    undeclared = replica.declare_bad_file_replicas(replicas_lst, reason=reason, issuer=issuer, status=BadFilesStatus.BAD, session=session)
+
```

Comment:
From a user perspective it is not clear what happens when a caller sets both `rse_id` and `rse`, possibly mismatching.
L103 reveals that `rse_id` takes always priority, but unless one inspects the code it is definitely not clear. Is this the intended behavior, or perhaps the call should fail? Would you mind clarify, at least with a comment on the function documentation?